test/index.coffee

```diff
@@ -24,3 +26,27 @@ Amen.describe "Haiku9 static-site generation", (context) ->
 
     context.test "Compile CoffeeScript files", ->
       assert yield exists "build/site.js"
+
+  context.test "Run the serve task", ->
+    context.test "Starts a local server on the default port", ->
+      command = join __dirname, "..", "bin", "h9"
+      child = childProcess.spawn(command, ["serve"])
+
+      # Give the server up to 5 seconds to spin up (but resolve immediately when you can)
+      start = now = new Date()
```

Comment:
Mostly because I am dumb, but why the `start = now = new Date()`? Are we setting both start and now to the same thing? Or is some kind of magic happening here?